doc/config.example.yaml

```diff
@@ -26,11 +26,11 @@ node:
   # Port on which we bind
   port:    2826    # 0xB0A, default value
   # Number of milliseconds to wait between retrying requests
-  retry_delay: 5000
+  retry_delay: 500
```

Comment:
I think 5 seconds was actually better.

Reply:
Will update.

Reply:
@Geod24 I'd like to confirm that the value(`5000` msecs) you suggested is right before applying it to the code. I think it's kind of long. Do you mean we should have 5 seconds for `retry_delay` and 5 seconds for `timeout`?

Reply:
@Geod24 I'd like to confirm that the value(5000 msecs) you suggested is right before applying it to the code. I think it's kind of long. Do you mean we should have 5 seconds for retry_delay and 5 seconds for timeout?

Reply:
Yes